css/index.css

```diff
@@ -143,4 +143,61 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+ 
+.topstuff {
+    display:inline-flex;
+    padding: 20px;
+}
+
+.bigimg {
+    margin-bottom:15px;
+    margin-top:10px;
+}
+
+.logo {
+    height: 45px;
+    margin-left: 540px;
+}
+.topnav {
+    padding: 15px;
+	/* margin-left: 100px; */
+
+}
+
+.anav {
+    font-weight: bold;
+    padding: 10px;
+    margin-left: 55px;
+    color: black;
+    text-decoration: none;
+    font-size: 13px;
+}
+
```

Comment:
Nice work changing only the color in the individual boxes and letting the parent take care of the rest of the styling.